draft-ietf-quic-http.md

```diff
@@ -305,10 +304,40 @@ encouraged to coordinate their experiments on the quic@ietf.org mailing list.
 
 ## Discovering an HTTP/3 Endpoint {#discovery}
 
+HTTP relies on the notion of an authoritative response: a response that has been
+determined by (or at the direction of) the origin server identified within the
+target URI to be the most appropriate response for that request given the state
+of the target resource at the time of response message origination.  Locating an
+authoritative server for an HTTP URL is discussed in Section 5.4 of
+{{!SEMANTICS}}.
+
+The "https" scheme associates authority with the ability of a server to use a
+private key associated with a certificate that the client considers to be
+trustworthy for the host identified by the authority component of the URL. If a
+server presents a certificate that verifiably applies to the host, along with
+proof that it controls the corresponding private key, then a client will accept
+a secured connection to that server as being authoritative for all origins with
+the same scheme and host.
+
+When an "https" URI is used within a context that calls for access to the
+indicated resource, a client MAY attempt access by resolving the host identifier
+to an IP address, establishing a QUIC connection to that address on the
+indicated port, and sending an HTTP/3 request message to the server over that
+secured connection containing the URI's identifying data.
+
+Connectivity problems (e.g., firewall blocking UDP) can result in QUIC
```

Comment:
```suggestion
Connectivity problems (e.g., blocking UDP) can result in QUIC
```
nit: I'd probably remove firewall, since lots of things can block UDP.